doc/user_manual/chInitialConditions.tex

```diff
@@ -19,7 +19,7 @@ \section{Tracking Parameters} \label{TraPar}
     \textbf{Keyword}    & \texttt{TRAC}\index{TRAC} &\\
     \textbf{Data lines} & 3 &\\
     \textbf{Format}     & Line 1: & \texttt{numl numlr napx amp(1) amp0 ird imc} \\
-                        &         & \texttt{niu(1) niu(2) numlcp numlmax} \\
+                        &         & \texttt{niu(1) niu(2) numlcp} \\
```

Comment:
11 characters? Do you mean 11 values? Nothing.
I seriously doubt this is even used by anyone, and the description doesn't match what the code actually did. The description is left commented out in the manual in any case.

Reply:
Yes, 11 values. The commented out description tells you nothing about where it was set tough, which means it is now just noise.

Also, the same goes for nde, and I thought it was easier to just mention it here rather than spam the issue tracker, but I can do that...

Reply:
I am not suggesting to put this in the issue tracker. That was for all your comments unrelated to the PR. Not sure what the best is here. This was always an optional input parameter that neither did what the manual described, nor was ever fully implemented in the code as far as I can tell.